### ISSUE TITLE
Examples contain a lot of non relevant code

### ISSUE THREAD

**Issue by MisteryX** [pyserial/pyserial#594]
Current examples contains a lot of complex code non relative to library. Why don`t you make basic examples with library calls only? It will help to developers to start to use library quickly without long time reading docs and try-mistake iterations.

**Comment by zsquareplusc** [comment]
yeah the example files in the repo are all more or less complex applications. but what would be a simple example? a bit of a problem is that there are so many protocols and devices, so that it is rare that a new developer can run the same example with his own setup.

**Comment by Kristine1975** [comment]
Maybe a loopback example?

**Comment by dj-fiorex** [comment]
i think that @Kristine1975 is right! It will be wonderful for the newcomers to this library to have an example integration with the multithreading package and the "new" async.io package
